modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets.adoc_{context}"]
+= Create a Secret object by using the web console
+
+As an administrator, you can create secrets using the web console. 
+
+. Navigate to *Workloads* → *Secrets*.
```

Comment:
Instead of using the icon, you should use "->". So:

```
. Navigate to *Workloads* -> *Secrets*.
```

Same for the next line